README.md

```diff
@@ -1,57 +1,147 @@
-# Project Name
+# [Spring PetClinic Sample Application](https://github.com/spring-projects/spring-petclinic) using a full set of Azure solutions
 
-(short, 1-3 sentenced, description of the project)
+* [Azure AppService](https://azure.microsoft.com/en-us/products/app-service/) for app hosting,
+* [Azure Database for MySQL](https://azure.microsoft.com/en-us/products/mysql/) for storage,
+* [Azure Monitor](https://azure.microsoft.com/en-us/products/monitor/)([Application Insights](https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview?tabs=net)) for monitoring and logging.
+* [Key Vault](https://azure.microsoft.com/en-us/products/key-vault/) for securing secret/password.
 
-## Features
+This application uses the [Azure Developer CLI (azd)](https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/overview)
+to get you up and running on Azure quickly.
 
-This project framework provides the following features:
+## Diff from the original Spring PetClinic 
+diff from [9ecdc111](https://github.com/spring-projects/spring-petclinic/commit/9ecdc1111e3da388a750ace41a125287d9620534)
```

Comment:
@wangmingliang-ms Do you think it'd be better if we move down to before the Security section? That way it reads more like a readme -- which assumes the user doesn't know much. Then we can explain the differences.

Reply:
I don't have preference for this, I will fix as you suggested. thanks.